packages/web/src/containers/earn-my-position-container/EarnMyPositionContainer.tsx

```diff
@@ -43,7 +43,7 @@ const EarnMyPositionContainer: React.FC<
 
     const { connected, connectAdenaClient, isSwitchNetwork, switchNetwork, account } = useWallet();
     const { updateTokenPrices } = useTokenData();
-    const { updatePositions, isFetchedPools, loading } = usePoolData();
+    const { updatePositions, isFetchedPools, loading: loadingPool } = usePoolData();
```

Comment:
What about `isLoading` or `isLoadingPool` instead of `loadingPool`?

Reply:
i will fix this